### PR TITLE
Updated the API URL and tweaked the way performer is imported.

### PR DESCRIPTION
**Pull request by darrenlevy**

### CODE REVIEW

File: DCM/DCMImportOperation.m [hunk 1]
```diff
@@ -233,6 +234,22 @@ - (void)importShow:(NSDictionary *)info
...
+We could add cast headshots later with:
+http://974d0d3e3fab692e5845-1fa162517f41f1b2710e3fd3bd0f08f4.r84.cf5.rackcdn.com/person_{{ PERSON_ID }}.png
+for thumbnail headshots (75pxX75px)
+or
+http://2292774aeb57f699998f-7c1ee3a3cf06785b3e2b618873b759ef.r47.cf5.rackcdn.com/person_{{ PERSON_ID }}.png
+for full size (400pxX400px)
+**/
+    NSDictionary *castDictionary = info[@"cast"];
+    for (id key in castDictionary) {
+        Performer *perf = [self performerFromName:[castDictionary objectForKey:key]];
```
Comment (benzado) [inline DCM/DCMImportOperation.m]: Any reason you didn't want to replace this `performerFromName:` with a lookup by ID?

Also, what are the values in the cast dictionary?

File: DCM/DCMImportOperation.m [hunk 2]
```diff
@@ -233,6 +234,22 @@ - (void)importShow:(NSDictionary *)info
             [show addPerformersObject:perf];
         }
     }
+
```
Comment (benzado) [inline DCM/DCMImportOperation.m]: Don't comment out code, just delete it! That's what what have git for.

**Comment by benzado** [comment]
Sorry I'm behind on this stuff, I can't do much tonight but I've carved out time tomorrow.

File: DCM/DCMImportOperation.m (hunk 1 above)
Comment (darrenlevy) [inline DCM/DCMImportOperation.m]: Ah, I didn't think of a lookup by ID. Good idea! Want me to write one?
The values in the cast dictionary are first and last name.

**Pull request merged**